### DISCUSSION TITLE
When performing stress testing, the following two errors occur when the concurrency is large

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#705] [Q&A] [answered]
@junoyoon 
When performing stress testing, the following two errors occur when the concurrency is large

`2018-11-28 01:19:37,364 ERROR Aborting thread: net.grinder.engine.process.ShutdownException: Thread has been shut down`
and
`net.grinder.plugin.http.TimeoutException: Read timed out`
why?

**Comment by junoyoon** [comment, marked as answer]
It's because the test thread does not have chance to handle the response.
Please use appropriate vuser count.
